tasks/kafka-broker/main.yml

```diff
@@ -48,6 +48,20 @@
   notify: restart kafka
   become: yes
 
+- name: Add Custom directive for the [Service] section (systemd)
+  ini_file:
+    path: "{{ kafka_broker_service_file }}"
+    section: Service
+    # Mind the quote in option and option ! We want it to be a quoted string in the target file
```

Comment:
we'll need to be careful when defining the options in meta_confluent with the quotes